src/DebugEngineHost.Stub/project.json

```diff
@@ -4,6 +4,7 @@
     "dnxcore50.app": {}
   },
   "dependencies": {
+    "MicroBuild.Core": "0.2.0",
```

Comment:
So it looks like FilesToSign is already part of DropFiles.targets, (which is included in MIDebugPackage.csproj). However, the final binaries do not appear to be signed.